toqito/nonlocal_games/extended_nonlocal_game.py

```diff
@@ -80,6 +80,13 @@ def __init__(self, prob_mat: np.ndarray, pred_mat: np.ndarray, reps: int = 1) ->
             self.pred_mat = pred_mat2
             self.reps = reps
 
+    def __get_game_dims(self):
```

Comment:
Add a docstring for this. I know this will not appear in the docs, but if we want to make changes to this in the future and you are unavailable, the docstring will be helpful.